### PR TITLE
Fix bug env:set command could not replace a string contains spaces

### PR DESCRIPTION
**Pull request by nguyentien98**
## About

- When you set a variable and its value contains the spaces. You couldn't do anything with **artisan**. The follow error will be show:
> The environment file is invalid: Dotenv values containing spaces must be surrounded by quotes.

- Now, if you set a varible with value contains the spaces. This value will be surrounded by quotes.

- If you run `php artisan config:cache` before using `env:set` command, this command can't set value for a existing key. Now it's fixed.

- If a key has a boolean value, `env:set` couldn't replace it. Now, it can.
## References

- N/A

### CODE REVIEW

File: src/Console/EnvSetCommand.php [hunk 1]
```diff
@@ -39,18 +39,20 @@ class EnvSetCommand extends Command
...
-        }
 
-        $this->setVariable($name, $newValue, $oldValue);
+            return;
+        }
 
+        $this->setVariable($name, $newValue);
         $this->info("Environment variable '$name' has been changed from '$oldValue' to '$newValue'");
+
+        return;
```
Comment (xuanquynh) [inline src/Console/EnvSetCommand.php]: Remove this return statement.

File: src/Console/EnvSetCommand.php [hunk 2]
```diff
@@ -60,11 +62,11 @@ public function handle()
...
+    public function setVariable($name, $value)
     {
-        $replaced = str_replace(
-            "$name=$oldValue",
-            "$name=$newValue",
+        $replaced = preg_replace(
+            "/$name=.*/",
+            "$name=$value",
             file_get_contents($envPath = $this->laravel->environmentFilePath())
         );
```
Comment (xuanquynh) [inline src/Console/EnvSetCommand.php]:
```php
return file_put_contents($envFilePath, $replaced);
```

File: src/Console/EnvSetCommand.php [hunk 3]
```diff
@@ -60,11 +62,11 @@ public function handle()
...
-    public function setVariable($name, $newValue, $oldValue)
+    public function setVariable($name, $value)
     {
-        $replaced = str_replace(
-            "$name=$oldValue",
-            "$name=$newValue",
+        $replaced = preg_replace(
+            "/$name=.*/",
+            "$name=$value",
             file_get_contents($envPath = $this->laravel->environmentFilePath())
```
Comment (xuanquynh) [inline src/Console/EnvSetCommand.php]: Change the variable name into `$envFilePath`.

File: src/Console/EnvSetCommand.php [hunk 4]
```diff
@@ -88,4 +90,33 @@ public function appendVariable($name, $value)
...
+    public function quote($string)
+    {
+        return strpos($string, ' ') !== false ? sprintf('"%s"', $string) : $string;
+    }
+
+    /**
+     * Get value of a key in .env file.
+     *
+     * @param  string  $key
+     * @return string
```
Comment (xuanquynh) [inline src/Console/EnvSetCommand.php]: This method returns `mixed`

**Comment by xuanquynh** [comment]
@nguyentien98 Thanks!

**Comment by nguyentien98** [comment]
You're welcome. :dancer:

**Pull request merged**